File: app/src/main/java/cn/invonate/ygoa3/Util/MailHolder.java
package cn.invonate.ygoa3.Util;


import java.util.List;

import javax.mail.Folder;
import javax.mail.internet.MimeMessage;

import cn.invonate.ygoa3.Entry.Mail;

/**
 * Created by liyangyang on 2017/10/26.
 */

public class MailHolder {
    public static List<MimeMessage> mails;
    public static List<Mail> mail_model;

    public static Folder folder;

}

File: app/src/main/java/cn/invonate/ygoa3/Util/MailHolder.kt
package cn.invonate.ygoa3.Util


import cn.invonate.ygoa3.Entry.Mail
import cn.invonate.ygoa3.Entry.MailNew
import java.util.*
import javax.mail.Folder
import javax.mail.internet.MimeMessage

/**
 * Created by liyangyang on 2017/10/26.
 */

object MailHolder {
    var mails: List<MimeMessage>? = null
    var mail_model: List<Mail>? = null

    var folder: Folder? = null

    var mailsBeans: ArrayList<MailNew.ResultBean.MailsBean>? = null

}
